example-apps/chatbot-rag-app/frontend/src/store/provider.tsx

```diff
@@ -233,7 +233,7 @@ export const thunkActions = {
         
               dispatch(actions.setStatus({ status: AppStatus.Done }))
             } else {
-              message += message && event.data.trim() === '' ? '\n' : event.data
```

Comment:
The issue here is that many times the stream returned by the LLM is just a space between words. That should be preserved literally in the frontend, I think.